javalin/src/main/java/io/javalin/config/PluginConfig.kt

```diff
@@ -33,6 +33,11 @@ class PluginConfig(private val pvt: PrivateConfig) {
     fun enableRedirectToLowercasePaths() = register(RedirectToLowercasePathPlugin())
     fun enableBasicAuth(username: String, password: String) = register(BasicAuthPlugin(username, password))
     fun enableSslRedirects() = register(SslRedirectPlugin())
-    fun enableCors(userConfig: Consumer<CorsPluginConfig>) = register(CorsPlugin(userConfig))
+    fun enableCors(userConfig: Consumer<CorsPluginConfig>) {
```

Comment:
Since the operation is additive now, it should probably be `addCors` or `addCorsRule`.

Reply:
Naming things is hard. I currently like `addCors` as the new name making it similiar to the old `addStaticFiles`. Will change it.